python/py-custom_inherit/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-custom_inherit
+version             2.4.1
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         Provides tools for inheriting docstrings in customizable ways.
+long_description    {*}${description}
+
+homepage            https://github.com/rsokl/custom_inherit
+
+checksums           rmd160  0aafd9e709150525a8cc9fc4ce47c01b948f564c \
+                    sha256  7052eb337bcce83551815264391cc4efc2bf70b295a3c52aba64f1ab57c3a8a2 \
+                    size    31193
+
+python.versions     38 39 310 311
+
+if {${name} ne ${subport}} {
+    python.pep517   yes
```

Comment:
this can/should be moved outside of the `if` statement that becomes then obsolete

Reply:
Thanks, done.